### ISSUE TITLE
Delayed sound

### ISSUE THREAD

**Issue by hedwigeisenbarth** [stephen/airsonos#380]
Hi Stephen,
very nice code, thank you for sharing. 
Just one thing: the sounds is super delayed, anything that I can do about that?
Thanks,
Hedwig

**Comment by EOEfrance** [comment]
Thanks for this Stephen. I'm having the same delayed issue that Hedwig is having. A video I play on Youtube is not synced up to its sound. Happens in Safari and Firefox.

Is there a fix?

**Comment by ErikAGriffin** [comment]
I don't believe there is way using this service to have the audio not be delayed.  A workaround if you are using video is to use players that use offsetting audio, like the [VLC player](https://www.videolan.org/).

You can even load youtube videos in VLC, and then use hotkeys ('F' and 'G') to adjust the audio until it is synced with the video.  Obviously not ideal, but works if you need to watch a movie or similar.  To open up YouTube in VLC you use 'Open Network Stream...' and just paste the youtube url

**Comment by jfredson** [comment]
Damn, this makes this totally unusable. Anybody know of any way to get the audio to sync up for an iTunes movie rental or figure out how to play an iTunes movie rental in VLC?

**Comment by philippe44** [comment]
You don't have control when the audio is played on the Sonos, it's just impossible in the UPnP standard
